### PR TITLE
change allowtransparency case

### PR DESCRIPTION
**Pull request by sergeysova**
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://htmlbook.ru/html/iframe/allowtransparency

image
image

```
"react": "^16.13.1"
```

### CODE REVIEW

File: types/react/index.d.ts [hunk 1]
```diff
@@ -2034,7 +2034,7 @@ declare namespace React {
     interface IframeHTMLAttributes<T> extends HTMLAttributes<T> {
         allow?: string;
         allowFullScreen?: boolean;
-        allowTransparency?: boolean;
+        allowtransparency?: boolean;
```
Comment (eps1lon) [inline types/react/index.d.ts:2037]: I think we just want to remove that attribute. It isn't a standard attribute as far as I can tell (https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element). We don't add types for non-standard attributes.

**Review by eps1lon — CHANGES_REQUESTED** [review]

File: types/react/index.d.ts (hunk 1 above)
Comment (sergeysova) [reply types/react/index.d.ts:2037]: I think its should be another PR

Comment (eps1lon) [reply types/react/index.d.ts:2037]: We don't include lowercase attributes either. Only their supported camel-cased versions. Neither applies here. I can open a separate on that removes it. But then this PR won't be accepted. Your choice.

Comment (sergeysova) [reply types/react/index.d.ts:2037]: Open a separate PR, please

Comment (eps1lon) [reply types/react/index.d.ts:2037]: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48605

**Pull request closed without merging**